### ISSUE TITLE
Target ES2020

### ISSUE THREAD

**Issue by oed** [decentralized-identity/did-jwt#208]
So that we can use [noble-secp256k1](https://github.com/paulmillr/noble-secp256k1) which is an order of magnitude smaller than elliptic.

**Comment by mirceanis** [comment]
would that be enough to use this in react native without too much setup?

**Comment by oed** [comment]
Not stale!

**Comment by sondreb** [comment]
Already using `@noble/secp256k1`, it is 100% without dependencies so works great in for example Angular. Not sure about React Native though...

**Comment by paulmillr** [comment]
React native supports bigints now.

**Comment by paulmillr** [comment]
I also suggest to replace:

```
    "@stablelib/ed25519": "^1.0.2",
    "@stablelib/random": "^1.0.1",
    "@stablelib/sha256": "^1.0.1",
    "@stablelib/x25519": "^1.0.2",
    "bech32": "^2.0.0",
    "js-sha3": "^0.8.0",
```

with `@noble/hashes` and `@noble/ed25519`, both of which passed independent security audit. `bech32` can be replaced with `@scure/base`, which also passed the audit.

The packages support tree shaking, so you can use just whatever is needed e.g. just sha3, not ripemd160.

**Comment by mirceanis** [comment]
this was fixed in #284
